### ISSUE TITLE
GO:0008276 | protein methyltransferase activity | PTN000378681

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#5585]
* **PTHR ID & PTN node:**

GO:0008276 | protein methyltransferase activity | IBA with MGI:3611450 , MGI:2684917 , MGI:1921569 , S000003890 , Q8WXB1 , S000000475 , S000004275 , Q96G04 , Q5VZV1 , Q9H867 , Q96AZ1 , S000001372 , PTN000378681 , MGI:3645330 , O95568
-- | -- | --

[GO:0008276](https://www.pombase.org/term/GO:0008276)	
[protein methyltransferase activity](https://www.pombase.org/term/GO:0008276)
[IBA](http://wiki.geneontology.org/index.php/Inferred_from_Biological_aspect_of_Ancestor_(IBA)) with [MGI:3611450](http://www.informatics.jax.org/accession/MGI:3611450) , [MGI:2684917](http://www.informatics.jax.org/accession/MGI:2684917) , [MGI:1921569](http://www.informatics.jax.org/accession/MGI:1921569) , [S000003890](https://www.yeastgenome.org/search?q=S000003890&is_quick=true) , [Q8WXB1](https://www.uniprot.org/uniprot/Q8WXB1) , [S000000475](https://www.yeastgenome.org/search?q=S000000475&is_quick=true) , [S000004275](https://www.yeastgenome.org/search?q=S000004275&is_quick=true) , [Q96G04](https://www.uniprot.org/uniprot/Q96G04) , [Q5VZV1](https://www.uniprot.org/uniprot/Q5VZV1) , [Q9H867](https://www.uniprot.org/uniprot/Q9H867) , [Q96AZ1](https://www.uniprot.org/uniprot/Q96AZ1) , [S000001372](https://www.yeastgenome.org/search?q=S000001372&is_quick=true) , [PTN000378681](http://www.pantherdb.org/panther/lookupId.jsp?id=PTN000378681) , [MGI:3645330](http://www.informatics.jax.org/accession/MGI:3645330) , [O95568](https://www.uniprot.org/uniprot/O95568)

* **Sequences with problematic annotation (ID + gene/protein name):**

https://www.pombase.org/gene/SPCC4G3.16 pombe rib2

* **Type of Issue: Erroneous source or erroneous propagation, or other issue**

* Add the label 'high priority' if needed. Generally high priority issues affect a lot of proteins, and annotations are incorrect (as opposed to just imprecise).

**Comment by marcfeuermann** [comment]
Updated

**Comment by ValWood** [comment]
Hi Marc,
I'm sorry, I made a mistake with this one. It's a tandem fusion protein and the N-terminal is indeed a protein methyltransferase based on ortholog to SGD RKM5
note to self, reannotate (see https://pubmed.ncbi.nlm.nih.gov/21460220/ )
v

**Comment by marcfeuermann** [comment]
Hi Val,
It is what I was thinking, but as it is within a duplication, I wondered if you had some information that I did not have. Better no annotation then a wrong annotation. No worries, I can undo the change 😊.
Cheers,
Marc.
